census/keras/trainer/model.py

```diff
@@ -16,14 +16,12 @@
 
 """Implements the Keras Sequential model."""
 
-import itertools
-
 import keras
 import pandas as pd
 from keras import backend as K
 from keras import layers, models
 from keras.utils import np_utils
-from keras.backend import relu, sigmoid
+from keras.backend import relu, softmax
```

Comment:
Good change, thx!

Reply:
@puneith: Do you see something else that should be changed here?